### PR TITLE
Initial system test

### PR DESCRIPTION
**Pull request by ArtInLines**
As discussed earlier - here's the initial integration of all the different components. I am sure there are still several bugs (starting with the fact, that there's no audio), but that was the goal after all.
The code is not yet connected with te UI, simply to make testing changes for everybody faster. I wanna connect it to the UI too (though before tomorrow I probably won't have the time to do so).
If you want, feel free to fix bugs on this branch.
I'm not sure whether we should already merge this pr, which is why I haven't requested anyone for review.

### CODE REVIEW

**Comment by MalteRichert** [comment]
Upon first glance, the Harmonised data look good to me. I suspected that there seems to be no audio because the volume is really low in the beginning but even listening for 2 min (at 1:03 length) and metering with super high gain didn't show any signal. Will investigate further tomorrow.
Still think that we should change volume normalisation to have a certain minimum that is way higher than 0 because very low volumes will just appear like silence.

File: src/main/java/app/StateManager.java [hunk 1]
```diff
@@ -227,43 +226,34 @@ public static void testSound(String[] args) {
...
 						? calcLineData(instrMap.getFeedbackReverb().get(), sonifiablePriceMap)
 						: null;
-				;
 				boolean[] onOffReverb = instrMap.getOnOffReverb().isPresent()
 						? calcRangeData(instrMap.getOnOffReverb().get(), sonifiablePriceMap)
 						: null;
-				;
-				double[] frequency = null; // TODO: The mapping doesn't have a frequency parameter (yet)
-				;
+				double[] frequency = instrMap.getCutoff().isPresent()
```
[the remark this answers is not in the document]
Comment (ArtInLines) [reply src/main/java/app/StateManager.java:233]: Ah that makes sense. Thanks 👍

**Comment by MalteRichert** [comment]
Audio should work now 🚀

**Comment by Rex2002** [comment]
@ArtInLines mein letzter Commit mit dem wav-export hat einen try-catch block drin, der die FileNotFoundException, die man da auffangen muss einfach als RuntimeException wirft, wie kann man das in das Error-Handling vom StateManager einbinden?

**Comment by ArtInLines** [comment]
@Rex2002 In app/ liegt eine Klasse namens "AppError". Wenn die geworfen wird, fängt das der StateManager und gibt die Message an die UI weiter

**Review by MalteRichert — APPROVED** [review]
This is ready to be merged into main. Further testing and bug-fixing will continue in a new branch

**Pull request merged**